packages/app-mobile/components/NoteBodyViewer/NoteBodyViewer.tsx

```diff
@@ -30,9 +30,9 @@ interface Props {
 export default function NoteBodyViewer(props: Props) {
 	const theme = themeStyle(props.themeId);
 
-	const webViewStyle: any = useMemo(() => {
-		return { backgroundColor: theme.backgroundColor };
-	}, [theme.backgroundColor]);
+	const webViewStyle = {
+		backgroundColor: 'transparent',
+	};
```

Comment:
If it never changes, it should be defined outside the NoteBodyViewer function

Reply:
Sure thing!

Does d2a9d0b look like a step in the right direction?

Reply:
Sorry, what I meant is that you need to move the constant outside the function body, to prevent unnecessary re-renders. So move `const webViewStyle = { backgroundColor: 'transparent', };` before `export default function NoteBodyViewer(props: Props)`. Please don't pass it as a prop as it's just a constant.

Reply:
Okay, can do.

So like 799cb27, just moved outside the body.